tslearn/docs/shapelets.rst

```diff
@@ -0,0 +1,82 @@
+.. _shapelets:
+
+Shapelets
+=========
+
+Shapelets are defined in [1]_ as "subsequences that are in some sense
+maximally representative of a class".
+Unformally, if we assume a binary classification setting, a shapelet is said
```

Comment:
Informally

Reply:
I would also remove "said":

"a shapelet is said discriminant" -> "a shapelet is discriminant"